### PR TITLE
Add label's operation spec

### PR DESCRIPTION
**Pull request by JackSie**

### CODE REVIEW

**Comment by rueian** [comment]
要 rebase 不然不能 merge

**Comment by JackSie** [comment]
Rebasing on this branch has been done.

File: spec/operations/label_operation/destroy_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+RSpec.describe LabelOperation::Destroy do
+
+  include_context 'label with color and title'
+
+  subject { LabelOperation::Destroy.new(@members[0], @project, @label) }
+
+  describe '#process' do
+    context 'when called' do
+
+      it 'destroys the channel' do
```
Comment (jason1122g) [inline spec/operations/label_operation/destroy_spec.rb]: label

File: spec/support/shared_contexts.rb [hunk 2]
```diff
@@ -89,3 +89,11 @@
...
   end
 end
+
+shared_context 'label with color and title' do
+  include_context 'project with members'
+  before(:example) do
+    @label = FactoryGirl.create(:label)
+    @label.project = @project
+  end
+end
```
Comment (jason1122g) [inline spec/support/shared_contexts.rb]: This is better
``` ruby
shared_context 'label with color title project and member' do
  include_context 'project with members'
  before(:example) do
    @label = FactoryGirl.build(:label)
    @label.project = @project
    @label.save
  end
end
```

**Comment by jason1122g** [comment]
ready to merge @rueian

**Pull request merged**